simplesat/sat/policy.py

```diff
@@ -43,6 +42,12 @@ def get_next_package_id(self, assignments, clauses):
         """
 
         decision_set = self._decision_set
```

Comment:
If `_decision_set` is not updated anymore between `get_next_package_id` calls, can we rename it to something else like `initial_decision_set` or something to mark more clearly its intent ?